include/sch/S_Object/S_ObjectNormalized.hxx

```diff
@@ -23,7 +23,7 @@ inline Point3 S_ObjectNormalized::support(const Vector3& v,int &LastFeature) con
   Vector3 vp(v);
   vp= v * mRot_; //ie : mRot.transpose*v (applying inverse transformation on the support vector)
   Scalar norm=vp.norm();
-  if (norm>0)
+  if (norm> sch::epsilon )
```

Comment:
The extra space is inconsistent with the surrounding code (at least with the change above)